src/gui/main-styles.ts

```diff
@@ -1110,11 +1110,13 @@ styles.registerStyle("main", () => {
 			bottom: 0,
 		},
 		".nav-bar-spacer": {
-			width: "2px",
+			width: "0px",
 			height: "24px",
 			"margin-left": "2px",
 			"margin-top": "10px",
-			"background-color": theme.navigation_border,
+			"border-color": theme.navigation_border,
```

Comment:
it's now two borders of 1px instead of one rectangle with 2px width. borders take up space so this should be the same, width-wise. 

although the height may have increased by 2px because of this, so that needs changing.